File: system-test/dotnet/Driver.Adapter/External/Erp/Client/Dtos/ExtProductDetailsResponse.cs
namespace Driver.Adapter.External.Erp.Client.Dtos;

public class ExtProductDetailsResponse
{
    public required string Id { get; set; }
    public required string Title { get; set; }
    public required string Description { get; set; }
    public decimal Price { get; set; }
    public required string Category { get; set; }
    public required string Brand { get; set; }
}

File: system-test/java/src/main/java/com/optivem/shop/testkit/driver/adapter/external/erp/client/dtos/ExtProductDetailsResponse.java
package com.optivem.shop.testkit.driver.adapter.external.erp.client.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ExtProductDetailsResponse {
    private String id;
    private String title;
    private String description;
    private BigDecimal price;
    private String category;
    private String brand;
}
